Packages/com.unity.inputsystem/InputSystem/Editor/ProjectWideActions/ProjectWideActionsAsset.cs

```diff
@@ -227,6 +193,31 @@ internal static bool CheckForDefaultUIActionMapChanges(InputActionAsset asset, I
         }
 
 #endif // UNITY_2023_2_OR_NEWER
+
+        // Returns the default UI action map as represented by the default template JSON.
```

Comment:
Moving this made it harder to review but not clear needed to be moved.

Reply:
Sorry that some code had to be moved but was difficult to piece tests together without relocating code to assemblies where it should run and where test utils could be referenced properly.

Reply:
This was an unnecessary move indeed, but will keep it like this since it moves private functions to the end. Will resolve this, feel free to reopen if I should really revert this.